config/get_hypre_version.cpp

```diff
@@ -18,12 +18,16 @@
 #define HYPRE_VERSION_STRING HYPRE_PACKAGE_VERSION
 #endif
 
+// Macros to expand a macro as a string
+#define STR_EXPAND(s) #s
+#define STR(s) STR_EXPAND(s)
```

Comment:
According to Google, we need to use the two-step process to properly expand the macro.